website/docs/troubleshooting/troubleshooting-podman-on-macos.md

```diff
@@ -271,3 +271,17 @@ Error: vm podman-machine-default not found: podman-machine-default: VM does not
 #### Additional resources
 
 - [Issue #9860](https://github.com/podman-desktop/podman-desktop/issues/9860)
+
+## A warning pop-up on macOS: running out of disk space
+
+On macOS, `.plist` files appended Podman Desktop data to the previous log files, which increased their size. Because of this, a warning pop-up stating `running out of disk space` appeared.
+
```

Comment:
question: I think the popup may not appear, I would say it's more about a general recommendation to check the size of the log files